.github/actions/spell-check/allow/code.txt

```diff
@@ -220,12 +226,21 @@ TABLETPC
 artanh
 arsinh
 arcosh
+roundf
 
 # Linux
 
 dbus
 anypass
+github
 gpg
+https
+ssh
+ubuntu
+workarounds
```

Comment:
This set I generally cary around, I suspect that one of the supplementary dictionaries also includes them, but...